spec/convection/model/cloudfile_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+require 'stringio'
+
+module Convection::Model
+  describe Cloudfile do
+    let(:template) do
+      Convection.template do
+        description 'EC2 VPC Test Template'
+
+        ec2_vpc 'TargetVPC' do
+          network '10.0.0.0'
+          subnet_length 24
+          enable_dns
+        end
+      end
+    end
+
+    describe 'stack' do
+      subject do
+        klass = Class.new
+        klass.extend(Convection::DSL::Cloudfile)
```

Comment:
`klass = Class.new(Convection::DSL::Cloudfile)` would've been equivalent to the two lines above.